smg-automotive/components-pkg
# feat: add google, save search, autoscout, motoscout icon

References:
[DM-1927](https://autoricardo.atlassian.net/browse/DM-1927)
[DM-1879](https://autoricardo.atlassian.net/browse/DM-1879)

## Motivation and context

We need:

- 3 icons for DM-1927 - Google icon, Motoscout24, Autoscout24
- 1 icon for DM-1879 - Save search

## Before

No relevant icons

## After

With icons

## How to test

Go to https://add-icons-components-pkg.branch.autoscout24.dev/?path=/story/foundations-icons--icons
Check following icons:
- AutoScout24Icon
- MotoScout24Icon
- GoogleIcon
- SaveSearchIcon

[DM-1927]: https://autoricardo.atlassian.net/browse/DM-1927?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DM-1879]: https://autoricardo.atlassian.net/browse/DM-1879?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
